tests/wpunit/AssertValidSchemaTest.php

```diff
@@ -284,7 +284,7 @@ public function testSchemaSupportsLazyLoadingTypes() {
 	}
 
 	public function testSchemaCallsLazyLoadingTypesWhenNeeded() {
-		$filter_calls = [];
+		$filter_calls = [ 'graphql_get_type' ];
```

Comment:
What's the reason for this change?

Reply:
see below comment: 

>>> Tests for me were returning them in different order. Always the same 3, but sometimes in different order.
>>> Perhaps that's something we should investigate more deeply instead of change this test

Reply:
I see that as a reason for the change below, but not for inserting this into the initial array. This change means that it will always look like `'graphql_get_type'` was called, even if it wasn't.

Reply:
oh shoot. Yes, I see 🤦🏻